mandos-go/json/parse/parseValue.go

```diff
@@ -141,6 +141,14 @@ func (p *Parser) parseString(obj oj.OJsonObject) (string, error) {
 	return str.Value, nil
 }
 
+func (p *Parser) parseBoolOrFalse(obj oj.OJsonObject) bool {
```

Comment:
The preferred pattern is to return an error not a bool.

Reply:
It will return false if you type "displayLogs": "true" for instance, instead of an error.
For when it's missing, the default is somewhere else.